### DISCUSSION TITLE
Adding community links to readme badges

### DISCUSSION THREAD

**Discussion by jankapunkt** [meteor/meteor#12228] [Feature Requests]
I've seen many popular repos that have directly linked their social media (follow on twitter) or their communities (slack, discord) in their badges. I think this may be a good practice for Meteor to guide people from GitHub to these respective other channels.

**Comment by Grubba27** [comment]
We recently updated our README.md. I guess it would look nice to add those badges there as well. Would you create the issue by explaining more about what your thought was? Maybe giving those examples.
I was in the mood to show as well our [star history]( https://star-history.com/#meteor/meteor&Date) but I do not know

## Star History

[![Star History Chart](https://api.star-history.com/svg?repos=meteor/meteor&type=Date)](https://star-history.com/#meteor/meteor&Date)


## Edit

For those who may want to add the Twitter badge here is an example using the meteor 
[![Twitter](https://img.shields.io/badge/twitter-%23007ACC.svg?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/meteorjs)

Another thing people may like is the maintainers' tab like in [tRPC](https://github.com/trpc/trpc#all-contributors-)

**Reply by jankapunkt** [reply]
What about the twitter with followers: 

![Twitter Follow](https://img.shields.io/twitter/follow/meteorjs?style=social)

some more interesting badges:

- repo stars: ![GitHub Repo stars](https://img.shields.io/github/stars/meteor/meteor?style=social) (seems redundant but think about preview links in social media that will contain this)
- license: ![GitHub](https://img.shields.io/github/license/meteor/meteor) (seems that the license is not autmatically readable?)
- node version in npm `meteor` package: ![node-current](https://img.shields.io/node/v/meteor) (seems to be not specified?)
- release tags ![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/meteor/meteor)
- meteor community discord: ![Discord](https://img.shields.io/discord/679410334292901888) (Admins must enable widgets)
- discussions: ![GitHub Discussions](https://img.shields.io/github/discussions/meteor/meteor)

**Reply by StorytellerCZ** [reply]
Repo starts might be interesting for the npm package readme, not so much in the main readme as people can just look up.
